driver/modern_bpf/programs/tail_called/events/syscall_dispatched_events/futex.bpf.c

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (C) 2023 The Falco Authors.
+ *
+ * This file is dual licensed under either the MIT or GPL 2. See MIT.txt
+ * or GPL2.txt for full copies of the license.
+ */
+
+#include <helpers/interfaces/fixed_size_event.h>
+
+/*=============================== ENTER EVENT ===========================*/
+
+SEC("tp_btf/sys_enter")
+int BPF_PROG(futex_e,
+        struct pt_regs *regs,
+        long id)
+{
+	struct ringbuf_struct ringbuf;
+	if(!ringbuf__reserve_space(&ringbuf, FUTEX_E_SIZE))
+	{
+		return 0;
+	}
+
+	ringbuf__store_event_header(&ringbuf, PPME_SYSCALL_FUTEX_E);
+
+	/*=============================== COLLECT PARAMETERS  ===========================*/
+
+	/* Parameter 1: addr (type: PT_UINT64) */
+	u64 addr = extract__syscall_argument(regs, 0);
+	ringbuf__store_u64(&ringbuf, addr);
+
+	/* Parameter 2: op (type: PT_ENUMFLAGS16) */
+	s32 op = (s32)extract__syscall_argument(regs, 1);
+	ringbuf__store_u16(&ringbuf, futex_op_to_scap((unsigned long)op));
+
+	/* Parameter 3: val (type: PT_UINT64) */
+	u64 val = extract__syscall_argument(regs, 2);
```

Comment:
another example of wasting bytes in our buffers here.. val is an uint32_t and we are pushing 8 bytes :( 
as far as I can see, we sometimes optimize other pieces of the libs but our events are not